modules/nf-core/nanoq/tests/main.nf.test.snap

```diff
@@ -0,0 +1,307 @@
+            {
+                "0": [
+                    [
+                        {
+                            "id": "test",
+                            "single_end": true
+                        },
+                        "test.stats:md5,5ab32af3352dfeca8268e10edf6e4dbe"
+                    ]
+                ],
+                "1": [
+                    [
+                        {
+                            "id": "test",
+                            "single_end": true
+                        },
+                        [
+                            "test.stats:md5,5ab32af3352dfeca8268e10edf6e4dbe",
+                            "test_nanoq.fastq.bz2:md5,b53cf14fd4eb5b16c459c41f03cc8a4b",
+                            "versions.yml:md5,7a40efe417ff7dbb9e91e9c1629a04e6"
+                        ]
+                    ]
+                ],
+                "2": [
+                    "versions.yml:md5,7a40efe417ff7dbb9e91e9c1629a04e6"
+                ],
+                "reads": [
+                    [
+                        {
+                            "id": "test",
+                            "single_end": true
+                        },
+                        [
+                            "test.stats:md5,5ab32af3352dfeca8268e10edf6e4dbe",
+                            "test_nanoq.fastq.bz2:md5,b53cf14fd4eb5b16c459c41f03cc8a4b",
+                            "versions.yml:md5,7a40efe417ff7dbb9e91e9c1629a04e6"
+                        ]
+                    ]
+                ],
+                "stats": [
+                    [
+                        {
+                            "id": "test",
+                            "single_end": true
+                        },
+                        "test.stats:md5,5ab32af3352dfeca8268e10edf6e4dbe"
+                    ]
+                ],
+                "versions": [
+                    "versions.yml:md5,7a40efe417ff7dbb9e91e9c1629a04e6"
+                ]
+            }
+        ],
+        "meta": {
+            "nf-test": "0.8.4",
+            "nextflow": "23.04.1"
+        },
+        "timestamp": "2024-06-28T10:54:04.660873"
+    },
+    "sarscov2 - nanopore - stub": {
+        "content": [
+            {
+                "0": [
+                    [
+                        {
+                            "id": "test",
+                            "single_end": true
+                        },
+                        "test.stats:md5,d41d8cd98f00b204e9800998ecf8427e"
+                    ]
+                ],
+                "1": [
+                    [
+                        {
+                            "id": "test",
+                            "single_end": true
+                        },
+                        [
+                            "test.stats:md5,d41d8cd98f00b204e9800998ecf8427e",
+                            "/Users/lili.li.2/modules/.nf-test/tests/9d4f60427672ca3da40466350f6c67ae/work/08/8e3658c9a4909104a7edb60966757c/test_nanoq.fastq.gz",
```

Comment:
There is an issue with nf-test and gzipped files. If you have a file ending .gz that isn't actually generated by gzip it gives this weirdness, where the filepath gets captured. We typically use `echo "" | gzip > file.gz` instead.

Reply:
`nanoq` can directly handle the compression.

Reply:
This issue is just in the stub, where it tries to go inside .gz files and can't manage it. This causes for some reason a full path to appear in the snapshot, which is causing your failures in the CI/CD.